src/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My ToDoList</title>
+</head>
+
+<body>
+    <div class="header">
```

Comment:
Following our Html & CSS guidelines rule number one, What do you think about using the header tag element for more semantic meaning instead of div? 🤔

Reply:
yes, you are right 👍  .. thank you for your feedback